server/src/main/java/org/jboss/as/server/operations/ServerProcessReloadHandler.java

```diff
@@ -159,6 +159,7 @@ protected ProcessReloadHandler.ReloadContext<RunningModeControl> initializeReloa
         if (serverConfig != null && !environment.getServerConfigurationFile().checkCanFindNewBootFile(serverConfig)) {
             throw ServerLogger.ROOT_LOGGER.serverConfigForReloadNotFound(serverConfig);
         }
+        ServerLogger.ROOT_LOGGER.warn("Reloading with config file " + serverConfig);
```

Comment:
@ehsavoie Please remove. I assume this is test cruft.

Reply:
correct